js/ui/tag_box.js

```diff
@@ -668,6 +668,8 @@ var TagBox = SelectBox.inherit({
         this._scrollContainer("end");
     },
 
+    _restoreInputText: noop,
```

Comment:
Looks dangerous to me!
The `_restoreInputText` is a [rather big function](https://github.com/DevExpress/DevExtreme/blob/ae140fe0cf1bdfdf9d2679e23dc69b0342d5c511/js/ui/select_box.js#L560-L579). This function is called not only in the `_focusOutHandler` function, but also in the `_toggleOpenState` function. 

What about possible side effects?

Reply:
It seems that _clearTextValue() quite enough for clearing / restoring input in tagBox.